src/pharmpy/model/statements.py

```diff
@@ -908,24 +908,32 @@ def get_n_connected(self, comp):
     def dosing_compartment(self):
         """The dosing compartment
 
-        A dosing compartment is a compartment that receives an input dose. Only one dose
-        compartment is supported.
+        A dosing compartment is a compartment that receives an input dose. Multiple
+        dose compartments are supported.
```

Comment:
Maybe it would be good here to write that we define the dose into central as the last dose compartment?

Reply:
Added description :)